.gitignore

```diff
@@ -13,3 +13,5 @@
 /Gemfile.lock
 /gemfiles/*.gemfile.lock
 .byebug_history
+
+/spec/conformance/metadata.zip
```

Comment:
This isn't having any effect because the file is also included in the diff here.

The intention was to ignore right?

Reply:
That must've gone wrong during a rebase, it was not meant to be included :( It's governed by the conformance tools terms of service and not open source.